app/views/admin/users/_index.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="adminUsersIndex">
+  <% @users.each do |user|  %>
+    <section id="user_<%= user.id %>">
+      <p><%= link_to user.name, admin_user_path(user) %></p>
+      <p><%= user.created_at.to_formatted_s(:long) %></p>
+      <p><%= user.role.to_s %></p>
+    </section>
+  <% end %>
+</div>
```

Comment:
Does this need to be a partial? As far as I know, this code is only needed in  `admin/users/index.html.erb`
(ignore me if I'm missing where else it's used)

Reply:
David, I would never ignore you 😄 Just fyi, I created it to eventually reuse with the `profile > users > show.html.erb` view because it's the same page with conditional logic for not showing the `Edit Profile` link to an admin. I just haven't had time to update that yet, but I will soon 👍